components/menu/Menu.js

```diff
@@ -162,6 +166,20 @@ const factory = (MenuItem) => {
         : undefined;
     }
 
+    handleKeyboardTrap = (event) => {
```

Comment:
@jodonnell I was thinking that extracting it to utils might make it less flexible b/c this way the keyboard trap can be made to cycle only in one direction. That's also why there's a bit of repetition b/c I didnt want to couple any of the functions. Also to extract would mean having to pass in the key code for different keys ('up','down', 'left', 'right', 'tab', 'shift tab') and wouldnt allow for a keyboard trap to listen to two different keys at the same time. Do you think it makes sense to try to allow for more flexibility that way or is it better to extract it?